### PR TITLE
Use material-table instead of antd for List Skill#2328

### LINKED ISSUE #2328: Use material-table instead of antd for Admin components
**Actual Behaviour**

Currently, antd library is present

**Expected Behaviour**

Use [material-table](https://github.com/mbrn/material-table) instead of antd for admin components, this will help us improve the performance by reducing the bundle size.

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by fragm3**
Fixes #2328, #2363  

Changes: 
- Use material-table instead of `antd` for List Skill
- Used styled-components
- Integrate Styled component in Admin Components #2363

Demo Link: https://pr-2339-fossasia-susi-web-chat.surge.sh

Screenshots of the change:

Screenshot 2019-06-21 at 6 51 30 PM

Screenshot 2019-06-21 at 6 51 43 PM

### CODE REVIEW

**Comment by fragm3** [comment]
Due to buggy in User tab, I have made the changes only for `List Skills` tab

**Comment by fragm3** [comment]
@samagragupta fixed

**Comment by harsh-jindal** [comment]
@fragm3 Fix this `Edit and delete options next to the search`
Screenshot 2019-06-22 at 12 40 46 PM

**Comment by fragm3** [comment]
@akshatnitd @harsh-jindal fixed.

Screenshots:
Screenshot 2019-06-22 at 8 54 45 PM

Screenshot 2019-06-22 at 8 54 53 PM

**Comment by fragm3** [comment]
@harsh-jindal fixed.

File: src/components/Admin/ListSkills/ListSkills.js [hunk 1]
```diff
@@ -42,12 +39,16 @@ const ActionSeparator = styled.span`
   margin-right: 0.313rem;
 `;
 
+const Bold = styled.span`
```
Comment (akshatnitd) [inline src/components/Admin/ListSkills/ListSkills.js]: Why wasn't `<b>` used ?

File: src/components/Admin/ListSkills/ListSkills.js [hunk 2]
```diff
@@ -417,198 +407,200 @@ class ListSkills extends React.Component {
...
-            </span>
-          );
-        },
+        field: 'language',
+        filtering: false,
       },
     ];
 
     return (
       <div className="tabs">
```
Comment (akshatnitd) [inline src/components/Admin/ListSkills/ListSkills.js]: Get rid of classnames completely.

**Comment by fragm3** [comment]
@akshatnitd Integrated styled components. Please review

**Pull request merged**